frontend/js/components/table/tableCell/generic/TableCellName.vue

```diff
@@ -1,30 +0,0 @@
-<template>
-  <a :href="editUrl"
-     v-if="!row.hasOwnProperty('deleted') && (row.edit)"
```

Comment:
Is this condition respected by the new table builder? When we link to edit, we should check if the record is not soft deleted.

Reply:
Was not taken care of but it is now!